www/scripts/codecheckerviewer/ListOfBugs.js

```diff
@@ -57,8 +58,21 @@ function (declare, Deferred, ObjectStore, Store, QueryResults, topic,
         function (reportDataList) {
           if (reportDataList instanceof RequestFailed)
             deferred.reject('Failed to get reports: ' + reportDataList.message);
-          else
+          else {
             deferred.resolve(that._formatItems(reportDataList));
+
+            if (query.reportFilters.length > 1)
+              hooks.bugFilteringChanged(query.reportFilters.length);
+            else {
+              var onlyFilter = query.reportFilters[0];
+              if (onlyFilter.filepath != "**" || onlyFilter.checkerId ||
```

Comment:
Use `!==` operator for equality check.
Moreover you use these condition in multiple places:
- https://github.com/whisperity/CodeChecker/blob/d403e6da6ed3cf7a089e0892af86b98c05ee823d/www/scripts/codecheckerviewer/ListOfBugs.js#L68-L70
- https://github.com/whisperity/CodeChecker/blob/d403e6da6ed3cf7a089e0892af86b98c05ee823d/www/scripts/codecheckerviewer/ListOfBugs.js#L125-L127